src/portfft/dispatcher/subgroup_dispatcher.hpp

```diff
@@ -355,27 +367,32 @@ PORTFFT_INLINE void subgroup_impl(const T* input, T* output, const T* input_imag
         }
       }
       sycl::group_barrier(global_data.it.get_group());
-      if (SubgroupSize != factor_sg || output_batch_interleaved) {
+      if (!store_directly_from_private) {
         global_data.log_dump_local("computed data in local memory:", loc_view, n_reals_per_wi * factor_sg);
         // store back all loaded batches at once.
-        if (!output_batch_interleaved) {
+        // data is batch interleaved in local
+        if (!is_output_batch_interleaved) {
           global_data.log_message_global(__func__,
                                          "storing transposed data from local to global memory (SubgroupSize != "
```

Comment:
Given that you have merged the packed / unpacked output codepath into one here,
this message does not hold true anymore

Reply:
updated in https://github.com/codeplaysoftware/portFFT/pull/142/commits/87dcefe0c86f7e0d159f3dbeac91e7f0327fbac8